invenio_theme_tugraz/config.py

```diff
@@ -63,29 +63,30 @@
 # COVER_TEMPLATE = 'invenio_theme_tugraz/accounts/accounts_base.html'
 """Cover page template for login and sign up pages."""
 
-SECURITY_LOGIN_USER_TEMPLATE = 'invenio_theme_tugraz/accounts/login_user.html'
+SECURITY_LOGIN_USER_TEMPLATE = "invenio_theme_tugraz/accounts/login_user.html"
 """Login template"""
 
-SECURITY_REGISTER_USER_TEMPLATE = \
-    'invenio_theme_tugraz/accounts/register_user.html'
+SECURITY_REGISTER_USER_TEMPLATE = "invenio_theme_tugraz/accounts/register_user.html"
 """Sigup template"""
 
 # Invenio-I18N
 # ============
 # See https://invenio-i18n.readthedocs.io/en/latest/configuration.html
-BABEL_DEFAULT_LOCALE = 'en'
+BABEL_DEFAULT_LOCALE = "en"
 # Default time zone
-BABEL_DEFAULT_TIMEZONE = 'Europe/Vienna'
+BABEL_DEFAULT_TIMEZONE = "Europe/Vienna"
 # Other supported languages (do not include BABEL_DEFAULT_LOCALE in list).
-I18N_LANGUAGES = [
-     ('de', _('German'))
- ]
+I18N_LANGUAGES = [("de", _("German"))]
 
 # Invenio-app-rdm
 # =============
 # See https://invenio-app-rdm.readthedocs.io/en/latest/configuration.html
-SEARCH_UI_HEADER_TEMPLATE = 'invenio_theme_tugraz/header.html'
+SEARCH_UI_HEADER_TEMPLATE = "invenio_theme_tugraz/header.html"
 """Search page's header template."""
 
-DEPOSITS_HEADER_TEMPLATE = 'invenio_theme_tugraz/header.html'
+DEPOSITS_HEADER_TEMPLATE = "invenio_theme_tugraz/header.html"
 """Deposits header page's template."""
+
+RECORDS_UI_ENDPOINTS["recid"].update(
+    template="invenio_theme_tugraz/record_landing_page.html"
+)
```

Comment:
revert all the changes from this file, it should be single-quotes, not double-quotes.

also consider adding comment for:

```suggestion
RECORDS_UI_ENDPOINTS["recid"].update(
    template="invenio_theme_tugraz/record_landing_page.html"
)
"""comments here"""
```